### DISCUSSION TITLE
How to set the BC for certain nodes?

### DISCUSSION THREAD

**Discussion by Yulongggggg** [idaholab/moose#28725] [Q&A Modules: Solid mechanics] [answered]
I have a 2D square mesh, and I wanted to set the displacement of a node at the bottom of the square, how can I implement it?

**Comment by GiudGiud** [comment]
Create a nodeset with just that node then use it in a dirichlet boundary conditions on the dipalacement with that nodeset as the boundary

**Reply by GiudGiud** [reply, marked as answer]
for example using this nodeset-generator
https://mooseframework.inl.gov/source/meshgenerators/BoundingBoxNodeSetGenerator.html
creating a tiny bounding box around the node
